doc/theory/glossary.tex

```diff
@@ -16,56 +68,254 @@
+	\subsection*{Filtered complex:}
+	A \textit{filtered complex} is a collection of simplicial or of cubical complexes $\{X(n)\}_{n \in \mathbb N}$ such that $X(n)$ is a subcomplex of $X(n+1)$ for each $n \geq 0$.
 	
-	\section{Geometry}
+	\subsection*{Simplexwise filtration:}
+	A \underline{filtered complex} such that $X(n+1)$ contains exactly one more simplex or elementary cube than $X(n)$. \par
+	The data of a simplexwise filtration is equivalent to a simplicial complex $X$ together with a total order $\leq$ on its simplices such that for each $y \in X$ the set $\{x \in X\ :\ x \leq y\}$ is a subcomplex of $X$.
 	
-	\paragraph{Metric space:} A  pair $(X, d)$ where $X$ is a set and $d$ is a function 
+	\subsection*{Persistence module:}
+	A \textit{persistence module} is a collection containing a $\Bbbk$-vector spaces $V(s)$ for each real number $s$ together with $\Bbbk$-linear maps $f_{st} : V(s) \to V(t)$, referred to as \textit{structure maps}, for each pair $s \leq t$, satisfying:
+	\begin{enumerate}
+		\item if $r \leq s \leq t$, then $f_{rt} = f_{st} \circ f_{rs}$ and
+		\item all but finitely many structure maps are isomorphisms.
+	\end{enumerate}
+	
+	A \textit{morphism of persistence modules} $F : V \to W$ is a collection of linear maps $F(s) : V(s) \to W(s)$ such that $F(t) \circ f_{st} = f_{st} \circ F(s)$ for each par of reals $s \leq t$.	We say that $F$ is an \textit{isomorphisms} if each $F(s)$ is.
+	
+	\subsection*{Persistent simplicial (co)homology:} 
+	Let $\{X(s)\}_{s \in \mathbb R} $ be a collection of ordered or directed simplicial complexes together with simplicial maps $f_{st} : X(s) \to X(t)$ for each pair $s \leq t$, such that if $r \leq s \leq t$, then $f_{rt} = f_{st} \circ f_{rs}$. Its \textit{persistent simplicial homology with $\Bbbk$-coefficients} is the persistence module
+	\begin{equation*}
+	H_*(X(s); \Bbbk)
+	\end{equation*}
+	with structure maps $H_*(f_{st}) : H_*(X(s); \Bbbk) \to H_*(X(t); \Bbbk)$ induced form the maps $f_{st.}$ In general, the collection constructed this way needs not me persistence module, but we restrict attention to the cases where it does.
+	
+	\textit{Persistence simplicial cohomology with $\Bbbk$-coefficients} is defined analogously.
+	
+	\subsection*{Vietoris-Rips complex and Vietoris-Rips homology:}
+	The \textit{Vietoris-Rips complex} of a \underline{finite metric space} $(X, d)$ is given by the following construction: let $s \geq 0$, define the {\underline{simplicial complex}} $VR_X(s)$ to have vertices the set $X$ and declare a subset $\{x_0, \dots, x_n\}$ of distinct points in $X$ to be a simplex if $d(x_i, x_j) \leq s$ for all $x_i, x_j$. 
```

Comment:
Minor suggestion: add an equation that outlines this for clarity?
`VR_X(s) = \{ x \subset X \mid d(x_i,x_j)\leq s,\, \forall x_i,x_j\in x\}` or something like this.